python/tests/test_mesh.py

```diff
@@ -25,9 +32,12 @@ def test_immutable(mesh):
         mesh.step = 1.0
 
 def test_volume_cartesian(cartesian_mesh):
+    assert cartesian_mesh.lower_boundary_condition ==  cartesian_mesh.boundary_type.periodic
+    assert cartesian_mesh.upper_boundary_condition ==  cartesian_mesh.boundary_type.periodic
```

Comment:
Once we change to strings, these can be:
```suggestion
    assert cartesian_mesh.lower_boundary_condition ==  "periodic"
    assert cartesian_mesh.upper_boundary_condition ==  "periodic"
```